### PR TITLE
Calculate tat for external samples

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
**This PR adds calculation of TAT for external samples in the status cases command**
External samples aren't marked as received/prepared/sequenced/delivered in LIMS and thus we approximate TAT to be the time from ordering to analysis upload. 

**How to test**:
1. install on stage of the hasta machine: `bash servers/resources/c.s.se/update-cg-stage.sh thisbranch`
1. activate stage: `us`
1. run following command: `cg status cases`

**Expected outcome**:
- [x] `cg` should show any cases with only external samples with TAT without parenthesis if it has it's analysis uploaded. 
- [x] TAT is calculated as time from ordering to analysis upload. 
- [x] Expected TAT (with parenthesis) for external samples is calculated as time from ordering to analysis completion + time from analysis completion to analysis upload. 

Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because it adds new but compatible functionality

### CODE REVIEW

**Comment by ingkebil** [comment]
Can you first resolve the conflicts? thans

**Comment by patrikgrenfeldt** [comment]
image
image
👍

File: scripts/calculate_avg_tat.sql [hunk 1]
```diff
@@ -29,4 +29,15 @@ join family_sample as f_s on f_s.`sample_id` = s.id
 join family as f on f.id = f_s.`family_id`
 join analysis as a on a.`family_id` = f.id
 where s.delivered_at is not null and a.uploaded_at is not null
-and DATEDIFF(s.delivered_at, a.uploaded_at) >= 0 AND DATEDIFF(s.delivered_at, a.uploaded_at) <= 31;
\ No newline at end of file
+and DATEDIFF(s.delivered_at, a.uploaded_at) >= 0 AND DATEDIFF(s.delivered_at, a.uploaded_at) <= 31;
+
+-- O_A external
+select avg(DATEDIFF(a.completed_at, f.ordered_at))
```
Comment (ingkebil) [inline scripts/calculate_avg_tat.sql]: Is this query here just to test the query out?

Comment (patrikgrenfeldt) [reply scripts/calculate_avg_tat.sql]: The code has some hard-coded values that has been calculated via the script. The script can be used if you want to verify the hard-coded values or if you want to calculate and update the values.

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**